tmux/record/record_ros_bag_Zed2i.yml

```diff
@@ -0,0 +1,122 @@
+
+# Pass command line options to tmux. Useful for specifying a different tmux.conf.
+# tmux_options: -f ~/.tmux.mac.conf
+tmux_options: -f <%= ENV["ANGEL_WORKSPACE_DIR"] %>/tmux/tmux.conf
+
+# Change the command to call tmux.  This can be used by derivatives/wrappers like byobu.
+# tmux_command: byobu
+
+# Specifies (by name or index) which window will be selected on project startup. If not set, the first window is used.
+# startup_window: editor
+
+# Specifies (by index) which pane of the specified window will be selected on project startup. If not set, the first pane is used.
+# startup_pane: 1
+
+# Controls whether the tmux session should be attached to automatically. Defaults to true.
+# attach: false
+
+windows:
+  # Connection with the HL2 app via HL2SS sensor data streaming and the
+  # ROS-TCP-Endpoint
+  # - sensor_input:
+  #     layout: even-vertical
+  #     panes:
+  #       - datahub: ros2 run ros_tcp_endpoint default_server_endpoint --ros-args
+  #           -r __ns:=${ROS_NAMESPACE}
+  #           -p ROS_IP:=0.0.0.0
+  #       - hl2ss_bridge: ros2 run angel_system_nodes hl2ss_ros_bridge --ros-args
+  #           -r __ns:=${ROS_NAMESPACE}
+  #           -p ip_addr:=${HL2_IP}
+  #           -p image_topic:=PVFramesBGR
+  #           -p image_ts_topic:=PVFramesBGR_TS
+  #           -p hand_pose_topic:=HandJointPoseData
+  #           -p audio_topic:=HeadsetAudioData
+  #           -p sm_topic:=SpatialMapData
+  #           -p head_pose_topic:=HeadsetPoseData
+  #           -p pv_width:=1280
+  #           -p pv_height:=720
+  #           -p pv_framerate:=30
+  #           -p sm_freq:=5
+  #           -p rm_depth_AHAT:=disable
+  #       # Visualize Images being output from the headset
+  #       - rqt_pv_images: rqt -s rqt_image_view/ImageView
+  #           --args ${ROS_NAMESPACE}/PVFramesBGR
+  #           --ros-args -p _image_transport:=raw 
+  - sensor_zed:
+      layout: even-vertical
+      panes:
+        # Read sensor input from a ZED 2i unit using the ROS2 wrapper.
+        # NOTE: The camera name given cannot start with a "/" as the value
+        # given is also used to prepend TF Frame IDs, which cannot begin with a
+        # slash.
+        - zed_bridge: ros2 launch zed_wrapper zed_camera.launch.py
+            camera_name:="${ROS_NAMESPACE#/}"/zed
+            camera_model:=zed2i
+            ros_params_override_path:=${ANGEL_WORKSPACE_DIR}/config/zed_params_override.yaml
+        # Enable sending ROS2 messages to receiving agents in an HoloLens2 app.
+        - datahub: ros2 run ros_tcp_endpoint default_server_endpoint  --ros-args
+            -r __ns:=${ROS_NAMESPACE}
+            -p ROS_IP:=0.0.0.0
+        # Trim out the timestamp of the image topic of choice.
+        - run_image_timestamp: ros2 run angel_system_nodes image_timestamp_relay --ros-args
+            -r __ns:=${ROS_NAMESPACE}
+            -p image_topic:="${ZED_IMAGE_TOPIC}"
+            -p output_topic:="${ZED_IMAGE_TS_TOPIC}"
```

Comment:
I would say that we don't need to create and record the timestamp messages. This node and output is trivial to run at system runtime.

Reply:
Removed the timestamp recording, thanks.

Reply:
I meant that the whole `run_image_timestamp` command could be removed, and then the `"${ROS_NAMESPACE}/${ZED_IMAGE_TS_TOPIC}"` topic being recorded may be removed.

Reply:
Translated intent to suggestions below.